test/spec/openapi/option.js

```diff
@@ -815,153 +1607,7 @@ test('move examples of parameters from component to media', async (t) => {
-
-  await fastify.ready()
-
-  const openapiObject = fastify.swagger()
-  const schema = openapiObject.paths['/'].post.requestBody.content['application/json'].schema
-
-  t.ok(schema)
-  t.ok(schema.properties.hello.examples)
-  t.same(schema.properties.hello.examples, ['hello', 'world'])
-})
-
-test('uses examples if has multiple numbers examples', async (t) => {
-  t.plan(3)
-  const fastify = Fastify()
-
-  await fastify.register(fastifySwagger, openapiOption)
-
-  const opts = {
-    schema: {
-      body: {
-        type: 'object',
-        required: ['hello'],
-        properties: {
-          hello: {
-            type: 'number',
-            examples: [1, 2]
-          }
-        }
-      }
-    }
-  }
-
-  fastify.post('/', opts, () => {})
-
-  await fastify.ready()
-
-  const openapiObject = fastify.swagger()
-  const schema = openapiObject.paths['/'].post.requestBody.content['application/json'].schema
-
-  t.ok(schema)
-  t.ok(schema.properties.hello.examples)
-  t.same(schema.properties.hello.examples, [1, 2])
-})
-
-test('uses examples if has multiple object examples', async (t) => {
-  t.plan(3)
-  const fastify = Fastify()
-
-  await fastify.register(fastifySwagger, openapiOption)
-
-  const opts = {
-    schema: {
-      body: {
-        type: 'object',
-        required: ['hello'],
-        properties: {
-          hello: {
-            type: 'object',
-            properties: {
-              lorem: {
-                type: 'string'
-              }
-            },
-            examples: [{ lorem: 'ipsum' }, { hello: 'world' }]
-          }
-        }
-      }
-    }
-  }
-
-  fastify.post('/', opts, () => {})
-
-  await fastify.ready()
-
-  const openapiObject = fastify.swagger()
-  const schema = openapiObject.paths['/'].post.requestBody.content['application/json'].schema
-
-  t.ok(schema)
-  t.ok(schema.properties.hello.examples)
-  t.same(schema.properties.hello.examples, [
-    { lorem: 'ipsum' },
-    { hello: 'world' }
-  ])
-})
-
-test('uses examples if has multiple array examples', async (t) => {
-  t.plan(3)
-  const fastify = Fastify()
-
-  await fastify.register(fastifySwagger, openapiOption)
-
-  const opts = {
-    schema: {
-      body: {
-        type: 'object',
-        required: ['hello'],
-        properties: {
-          hello: {
-            type: 'array',
-            items: {
-              type: 'string'
-            },
-            examples: [['a', 'b', 'c'], ['d', 'f', 'g']]
-          }
-        }
-      }
-    }
-  }
-
-  fastify.post('/', opts, () => {})
-
-  await fastify.ready()
-
-  const openapiObject = fastify.swagger()
-  const schema = openapiObject.paths['/'].post.requestBody.content['application/json'].schema
-
-  t.ok(schema)
-  t.ok(schema.properties.hello.examples)
-  t.same(schema.properties.hello.examples, [
-    ['a', 'b', 'c'],
-    ['d', 'f', 'g']
-  ])
-})
-
-test('uses examples if has property required in body', async (t) => {
+test('marks request body as required', async (t) => {
```

Comment:
This test had a wrong description, I fixed it since it mentions examples, but doesn't test anything related to them.

Originally introduced in https://github.com/fastify/fastify-swagger/pull/541